clang/test/CIR/CodeGen/call-extra-attrs.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -O2 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s -check-prefix=CIR
+
+__attribute__((nothrow))
+int s0(int a, int b) {
+  int x = a + b;
+  return x;
+}
+
+__attribute__((noinline))
+int s1(int a, int b) {
+  return s0(a,b);
+}
+
+int s2(int a, int b) {
+  return s1(a, b);
+}
+
+// CIR: #fn_attr = #cir<extra({nothrow = #cir.nothrow})>
+// CIR: #fn_attr1 = #cir<extra({inline = #cir.inline<no>, nothrow = #cir.nothrow})>
+
+// CIR: cir.func @_Z2s0ii(%{{.*}}, %{{.*}}) -> {{.*}} extra(#fn_attr)
+// CIR: cir.func @_Z2s1ii(%{{.*}}, %{{.*}}) -> {{.*}} extra(#fn_attr1)
+
+// CIR: cir.call @_Z2s0ii(%{{.*}}, %{{.*}}) : {{.*}}, {{.*}}) -> {{.*}} extra(#fn_attr)
+// CIR: cir.call @_Z2s1ii(%{{.*}}, %{{.*}}) : ({{.*}}, {{.*}}) -> {{.*}} loc(#loc{{.*}})
```

Comment:
I added `loc` because I want to explicitly check that there's no extra_attributes being printed here for the second call. Is there any other way of doing that?

Reply:
Gotcha. For that you might want to use `CIR-NOT: ...`, see https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-not-directive